images/dockerregistry/Dockerfile

```diff
@@ -6,20 +6,21 @@
 #
 FROM openshift/origin-base
 
-RUN INSTALL_PKGS="origin-dockerregistry" \
- && yum install -y ${INSTALL_PKGS} \
- && rpm -V ${INSTALL_PKGS} \
- && yum clean all
-
 COPY config.yml ${REGISTRY_CONFIGURATION_PATH}
```

Comment:
So this moved because that's how it is in dist-git. I didn't bother updating those images.